examples/bulk.php

```diff
@@ -68,7 +68,7 @@ function toJSON(object $document): string
                 ['x' => 10], // Document
             ],
         ],
-    ]
+    ],
```

Comment:
This change is responsible for most of the diff in this PR: a trailing comma after the last argument in multi-line method calls is now required. This is consistent with the rules already in place for multi-line arrays.